vdirel.el

```diff
@@ -204,6 +205,11 @@ CANDIDATE is ignored."
                      (helm-marked-candidates)
                      ", ")))
 
+(defun vdirel--open-file (candidate)
+  "Open file assosiated to CANDIDATE."
```

Comment:
can you please rephrase this docstring so it clearly says that `CANDIDATE` is ignored? You can copy/paste from `vdirel--helm-insert-contact-email`.